Dispatcher/Controller/HeaderViewController.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class HeaderViewController: UIViewController {
```

Comment:
the header is not a ViewController
its a UI Component not a screen

use the navigation header that is used in the navigation controller